nixos/modules/services/misc/nix-gc.nix

```
@@ -50,11 +82,18 @@ in
 
   config = {
 
-    systemd.services.nix-gc =
-      { description = "Nix Garbage Collector";
-        script = "exec ${config.nix.package.out}/bin/nix-collect-garbage ${cfg.options}";
-        startAt = optional cfg.automatic cfg.dates;
+    systemd.services.nix-gc = {
+      description = "Nix Garbage Collector";
+      script = "exec ${config.nix.package.out}/bin/nix-collect-garbage ${cfg.options}";
```

I don't think we need a wrapper script here:

```suggestion
      serviceConfig.ExecStart = "${config.nix.package.out}/bin/nix-collect-garbage ${cfg.options}";
```

---

I didn't touch this part. But if you want it in this merge request I can commit your suggestion.

---

Respectfully disagree with the suggestion. This has been done in 2013 and decided against in 65277727e89ef7cd8657eca1c0fda4c2892d3fea.

Additional arguments:
- *don't fix it if it ain't broken*
- unity of subject-matter: this PR is about the timer, not about tweaking the `ExecStart` value. This should be PRd and argued separately.

---

> this PR is about the timer, not about tweaking the ExecStart value. This should be PRd and argued separately.

Nixpkgs is so fast changing and I usually also just write comments when I see things which should be changed even if they are not touched in this PR to just fix it while we touch it.